fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingView.kt

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding.view
+
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.mozilla.fenix.nimbus.OnboardingPanel
+import org.mozilla.fenix.onboarding.OnboardingState
+import org.mozilla.fenix.onboarding.interactor.OnboardingInteractor
+import org.mozilla.fenix.nimbus.Onboarding as OnboardingConfig
+
+/**
+ * Shows a list of onboarding cards.
+ */
+class OnboardingView(
```

Comment:
this file seems straightforward. I'll be curious to see in the subsequent parts where/how `update()` is being called.

It all seems scalable though, and the refactor makes it simple to derive a Compose analog, long term, from this.

Reply:
This currently serves to update the list adapter when we consume the store and dispatch the new state. I don't think we will need this once we remove the recycler view.